pipelines/h5c/rtp/v1/stage_1_task_scripts/do_LIBRARIAN_XRFI.sh

```diff
@@ -18,21 +18,30 @@ if [ "${upload_to_librarian}" == "True" ]; then
     if [ "${librarian_xrfi}" == "True" ]; then
         # get the integer portion of the JD
         jd=$(get_int_jd ${fn})
+        decimal_jd=$(get_jd ${fn})
 
         # get xrfi folder
         xrfi_folder=`echo ${fn%.sum.uvh5}.xrfi`
         xrfi_stage_1_folder=`echo ${fn%.sum.uvh5}.stage_1_xrfi`
-        compressed_file=`echo ${fn%.sum.uvh5}.stage_1_xrfi.tar.gz`
+        compressed_file=`echo zen.${jd}.stage_1_xrfi.tar.gz`
         if [ -d "${xrfi_folder}" ]; then
-            echo tar -czfv ${compressed_file} ${xrfi_folder}
-            tar -czfv ${compressed_file} ${xrfi_folder}
-            echo librarian upload local-rtp ${compressed_file} ${jd}/${xrfi_folder%.xrfi}.stage_1_xrfi.tar.gz
-            librarian upload local-rtp ${compressed_file} ${jd}/${xrfi_folder%.xrfi}.stage_1_xrfi.tar.gz
+            echo tar -czfv ${compressed_file} zen.${jd}.*.xrfi
+            tar -czfv ${compressed_file} zen.${jd}.*.xrfi
+            echo librarian upload local-rtp ${compressed_file} zen.${jd}.stage_1_xrfi.tar.gz
+            librarian upload local-rtp ${compressed_file} zen.${jd}.stage_1_xrfi.tar.gz
         elif [ -d "${xrfi_stage_1_folder}" ]; then # if it has already been renamed
-            echo tar -czfv ${compressed_file} ${xrfi_stage_1_folder}
-            tar -czfv ${compressed_file} ${xrfi_stage_1_folder}
-            echo librarian upload local-rtp ${compressed_file} ${jd}/${xrfi_stage_1_folder}.tar.gz
-            librarian upload local-rtp ${compressed_file} ${jd}/${xrfi_stage_1_folder}.tar.gz
+            echo tar -czfv ${compressed_file} zen.${jd}.*.stage_1_xrfi
+            tar -czfv ${compressed_file} zen.${jd}.*.stage_1_xrfi
+            echo librarian upload local-rtp ${compressed_file} zen.${jd}.stage_1_xrfi.tar.gz
+            librarian upload local-rtp ${compressed_file} zen.${jd}.stage_1_xrfi.tar.gz
+            
+        # upload all thresholded flags files        
+        compressed_threshold_file=`echo zen.${jd}.stage_1_threshold_flags.tar.gz`
```

Comment:
This is just a few files per night, so I don't see a point of compressing them.

Reply:
I am leaning towards leaving it here for consistency, or you prefer just uploading them individually as we did originally?

Reply:
I prefer uploading them individually, since that's consistent with what we've done in the past.